content/trainings/_template.mdx

```diff
@@ -6,6 +6,8 @@ image: /images/example.png
 links:
   - url: https://example.com
     label: Website
+tags:
```

Comment:
suggestion: adding a comment for guidance to the template suggesting main tags, suggestion for trainings

```md
# for easy filtering please choose 1-4 tags:
# set type of training: coding, design
# set language/tool: python, javascript/typescript, c#, blender, rhino
# set free/paid: free, paid
# set mode: in-person, online, hybrid
```

Reply:
Yeah, that is actually not a bad idea to make it two-fold.
I'll update the PR to reflect that.